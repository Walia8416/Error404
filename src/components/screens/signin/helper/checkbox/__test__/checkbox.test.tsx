import {act, create} from 'react-test-renderer';

import Checkbox from '../CheckBox';
import React from 'react';

describe('Checkbox render', () => {
  it('initial render', () => {
    const checkbox = create(<Checkbox setState={() => jest.fn()} />);
    const snap = checkbox.toJSON();
    act(() => snap.children[0].props.onClick());
    expect(snap).toMatchSnapshot();
    const btn = checkbox.root.findByProps({testID: 'checkboxtrue'}).props;
    act(() => btn.onPress());
  });
});
