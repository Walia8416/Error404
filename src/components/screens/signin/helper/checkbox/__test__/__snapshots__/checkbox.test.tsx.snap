// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox render initial render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#C4D7E7",
      "borderWidth": 1,
      "height": 16,
      "justifyContent": "center",
      "marginRight": 0,
      "width": 16,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 16,
        "opacity": 1,
        "width": 16,
      }
    }
    testID="checkboxfalse"
  />
</View>
`;
